tests/rule_based_profiler/conftest.py

```diff
@@ -2,13 +2,16 @@
 
 import pytest
 
-# TODO: <Alex>ALEX -- We need to add tests involving "SemanticDomainTypes" (not only "StructuredDomainTypes").</Alex>
+# TODO: <Alex>ALEX -- We need to add tests involving "MetricDomainTypes".</Alex>
```

Comment:
Could you give an example?

Reply:
@talagluck I deleted these comments (based on our meeting this morning).  Thanks!